pkg/utils/utils.go

```diff
@@ -0,0 +1,43 @@
+package utils
```

Comment:
Please see https://blog.golang.org/package-names

`Avoid meaningless package names. Packages named util, common, or misc provide clients with no sense of what the package contains. This makes it harder for clients to use the package and makes it harder for maintainers to keep the package focused. Over time, they accumulate dependencies that can make compilation significantly and unnecessarily slower, especially in large programs. And since such package names are generic, they are more likely to collide with other packages imported by client code, forcing clients to invent names to distinguish them.`

One could argue the same for `models`, but it certainly happens less commonly. I feel like this API could be located in `hubbub`, or a `provider` package, where the models sit. Thoughts?

Reply:
I agree with you, thank you for notice. I got rid of 'utils' package - replaced it to 'triage' package. Should I replace 'models' code to 'provider' package?